ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -190,6 +192,67 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.ability({ id: '788B', source: 'Hegemone', capture: false }),
       response: Responses.moveAway(),
     },
+    {
+      id: 'P6S Predation Debuff Collect',
+      // CF7 Glossal Resistance Down (Snake Icon)
+      // CF8 Chelic Resistance Down (Wing Icon)
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['CF7', 'CF8'] }),
+      condition: Conditions.targetIsYou(),
+      run: (data, matches) => data.predationDebuff = matches.effectId,
+    },
+    {
+      id: 'P6S Predation Bait Order',
+      // Using Aetheronecrosis (CF9)
+      // These come out as 20s, 16s, 12s, or 8s
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'CF9' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 0.1,
+      durationSeconds: (_data, matches) => {
+        const duration = parseFloat(matches.duration);
+        return duration === 20 ? duration - 0.1 : duration + 16 - 0.1;
+      },
+      infoText: (data, matches, output) => {
+        const AetheronecrosisMap: { [duration: number]: string } = {
+          20.00: output.firstBait!(),
+          8.00: output.secondBait!(),
+          12.00: output.thirdBait!(),
+          16.00: output.fourthBait!(),
```

Comment:
I am not sure if everybody is doing this, but that's definitely the strat that I used too.  Do you think it'd be worth calling "bait near" or something when it's your turn to, after the aoes go off?

Reply:
Yea, I was going to write an in at the time to go in and then out after the 4s/hit, but never got around to it.

Reply:
Regarding strat, I think it's only possible to do baits in this order due to vulnerability up.

Reply:
I have added an in trigger and out trigger.

In trigger for first bait is based on the Dual Predation cast time, but realistically they could be in as soon as they get their debuff.

The other triggers are based on the hits, which means it is safe for the player to move in without getting aoe'd by the previous predation. This comes at a risk that the trigger will still fire if the player has a vulnerability up from making a mistake...